packages/mobile/locales/en-US/translation.json

```diff
@@ -496,6 +496,7 @@
   "helpFindAccount": "Help your contact find their Account Address",
   "close": "Close",
   "home": "Home",
+  "dappsExplorer": "dAPP Explorer",
```

Comment:
I think this text should be in the `base` directory as well?

Reply:
My bad, I think we can remove this one :)